PRIVACY.md

```diff
@@ -0,0 +1,93 @@
+The types of personal data collected may include (but are not limited to):
+
+* **Identifiers:** Unique Online Identifiers (e.g., User ID, Client ID, Cookie ID).
+* **Internet or Other Electronic Network Activity Information:** Browsing history, pages viewed, time spent on pages, click patterns, referring websites (Traffic Sources/Referrers), and search terms.
+* **Device and Usage Data:** Browser type, operating system (Platform), device type, screen size, IP address (or anonymized IP address), and language settings.
+
+## Using (Processing) Your Personal Data
+
+Konflux uses the personal data you provide to:
+
+* To analyze website traffic and usage patterns (Website Analytics).
+* To improve and optimize the performance and content of our website.
+* To understand user journeys and how users interact with our features.
+
+## Sharing Your Personal Data
+
+Unless you consent, Konflux will never process or share the personal data you provide to us except as described below.  
+Konflux may disclose your personal data to third parties under any of the following circumstances:
+
+* As required by law (such as responding to a valid subpoena, warrant, audit, or agency action, or to prevent fraud).  
+* For research activities, including the production of statistical reports (such aggregated information is used to describe our services and is not used to contact the subjects of the report).
+
+## Cookies and Other Browser Information
+
+Konflux's online services automatically capture IP addresses. We use IP addresses to help diagnose problems with our servers, to administer our website, and to help ensure the security of your interaction with our services. Your IP address is used to help identify you and your location, in order to provide you data and content from our services as quickly as possible. It is in Konflux’s legitimate business interest to maximize the efficiency and effectiveness of its services for all users.  
+As part of offering and providing customizable and personalized services, Konflux uses cookies to store and sometimes track information about you. A cookie is a small amount of data that is sent to your browser from a Web server and stored on your computer's hard drive.
+Generally, we use cookies to:
+
+2. Estimate audience size. Each browser accessing Konflux is given a unique cookie that is used to determine the extent of repeat usage and usage by a registered user versus by an unregistered user.  
+4. Measure certain traffic patterns, which areas of Konflux's network of websites you have visited, and your visiting patterns in the aggregate. We use this research to understand how our user's habits are similar or different from one another so that we can make each new experience on Konflux website a better one. We may use this information to better personalize the content, banners, and promotions you and other users will see on our sites.
```

Comment:
I would remove "promotions" or rewrite this last part unless there is plan to run ads on the website. It kinda feel weird on a community website.